janusgraph-core/src/main/java/org/janusgraph/core/schema/SchemaInspector.java

```diff
@@ -83,7 +84,27 @@
      * @see PropertyKey
      */
     default PropertyKey getOrCreatePropertyKey(String name, Object value) {
-       return getOrCreatePropertyKey(name);
+        return getOrCreatePropertyKey(name);
+    }
+
+    /**
+     * Returns the property key with the given name. If automatic type making is enabled, it will make the property key
+     * using the configured default type maker if a key with the given name does not exist. Ff automatic type making is enabled,
```

Comment:
You can use github suggestions for this. This is really cool.

Reply:
```suggestion
     * using the configured default type maker if a key with the given name does not exist. If automatic type making is enabled,
```

Reply:
As I understand commits from github suggestions will not be signed and DCO will complain about it. Also, I would prefer squashing commits which just resolve comments in PR into a single commit instead of creating additional commit like `Fix PR comments`. May the github suggestions be squashed into your commit?

Reply:
I think it is better to see what the different is.

Reply:
You are right but those commits will not be signed. Nevertheless the squash will be needed. 
We can also see the changes by clicking on `force-pushed` to see changes.